.circleci/config.yml

```diff
@@ -28,6 +31,18 @@ jobs:
             git tag -a v$(cat version.txt) -m "Release version $(cat version.txt)"
             git push --tags origin
             git push origin master
+  build_client:
+    executor: client-builder
+    working_directory: ~/ara/client
+    steps:
+      - checkout:
+          path: ~/ara
+      - run:
+          name: Prepare client
```

Comment:
`Install client's dependencies` seems more accurate to me 🤔

Reply:
I agree :)